src/internal/m365/collection/site/backup_test.go

```diff
@@ -22,6 +29,269 @@ import (
+			expectMetadataColls:  1,
+			canUsePreviousBackup: true,
+		},
+		{
+			name:                 "with error",
+			mock:                 siteMock.NewListHandler(siteMock.StubLists("one"), siteID, errors.New("some error")),
+			expectErr:            require.Error,
+			expectColls:          0,
+			expectNewColls:       0,
+			expectMetadataColls:  0,
+			canUsePreviousBackup: false,
+		},
+	}
+	for _, test := range table {
+		suite.Run(test.name, func() {
+			ctx, flush := tester.NewContext(t)
+			defer flush()
+
+			ac, err := api.NewClient(
+				suite.creds,
+				control.DefaultOptions(),
+				count.New())
+			require.NoError(t, err, clues.ToCore(err))
+
+			bpc := inject.BackupProducerConfig{
+				LastBackupVersion: version.NoBackup,
+				Options:           control.DefaultOptions(),
+				ProtectedResource: mock.NewProvider(siteID, siteID),
+			}
+
+			cs, canUsePreviousBackup, err := CollectLists(
+				ctx,
+				test.mock,
+				bpc,
+				ac,
+				suite.creds.AzureTenantID,
+				sel.Lists(selectors.Any())[0],
+				statusUpdater,
+				count.New(),
+				fault.New(false))
+
+			test.expectErr(t, err, clues.ToCore(err))
+			assert.Len(t, cs, test.expectColls, "number of collections")
+			assert.Equal(t, test.canUsePreviousBackup, canUsePreviousBackup)
+
+			newStates, metadatas := 0, 0
+			for _, c := range cs {
+				if c.FullPath() != nil && c.FullPath().Service() == path.SharePointMetadataService {
+					metadatas++
+					continue
+				}
+
+				if c.State() == data.NewState {
+					newStates++
+				}
+			}
+
+			assert.Equal(t, test.expectNewColls, newStates, "new collections")
+			assert.Equal(t, test.expectMetadataColls, metadatas, "metadata collections")
+		})
+	}
+}
+
+func (suite *SharePointBackupUnitSuite) TestCollectLists_incremental() {
+	t := suite.T()
+
+	var (
+		statusUpdater = func(*support.ControllerOperationStatus) {}
+		siteID        = tconfig.M365SiteID(t)
+		sel           = selectors.NewSharePointBackup([]string{siteID})
+	)
+
+	ac, err := api.NewClient(
+		suite.creds,
+		control.DefaultOptions(),
+		count.New())
+	require.NoError(t, err, clues.ToCore(err))
+
+	listPathOne, err := path.Build(
+		suite.creds.AzureTenantID,
+		siteID,
+		path.SharePointService,
+		path.ListsCategory,
+		false,
+		"one")
+	require.NoError(suite.T(), err, clues.ToCore(err))
+
+	listPathThree, err := path.Build(
+		suite.creds.AzureTenantID,
+		siteID,
+		path.SharePointService,
+		path.ListsCategory,
+		false,
+		"three")
+	require.NoError(suite.T(), err, clues.ToCore(err))
+
+	table := []struct {
+		name                string
+		lists               []models.Listable
+		deltaPaths          metadata.DeltaPaths
+		expectErr           require.ErrorAssertionFunc
+		expectColls         int
+		expectNewColls      int
+		expectNotMovedColls int
+		expectMetadataColls int
+		expectTombstoneCols int
+	}{
+		{
+			name:  "one list",
+			lists: siteMock.StubLists("one"),
+			deltaPaths: metadata.DeltaPaths{
+				"one": {
+					Delta: "one",
```

Comment:
`Delta` would not really be used right? Given lists don't support delta. Can we remove it to avoid confusion?